modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test/src/testIntegration/java/com/liferay/site/initializer/extender/internal/test/BundleSiteInitializerTest.java

```diff
@@ -357,6 +360,95 @@ public void testInitializeFromFile() throws Exception {
 		}
 	}
 
+	private void _assertAccountGroupAssignments(
+		AdminAccountGroup adminAccountGroup, int accountGroupsCount) {
+
+		Assert.assertEquals(
+			accountGroupsCount,
+			_accountGroupRelLocalService.
+				getAccountGroupRelsCountByAccountGroupId(
+					adminAccountGroup.getId()));
+	}
+
+	private void _assertAccountGroups1() throws Exception {
+		AdminAccountGroupResource.Builder accountGroupResourceBuilder =
+			_adminAccountGroupResourcefactory.create();
+
+		AdminAccountGroupResource accountGroupResource =
+			accountGroupResourceBuilder.user(
+				_serviceContext.fetchUser()
+			).build();
+
+		AdminAccountGroup accountGroup =
+			accountGroupResource.getAccountGroupByExternalReferenceCode(
+				"TESTACCOUNTGROUP1");
+
+		Assert.assertNotNull(accountGroup);
+		Assert.assertEquals("Test Account Group 1", accountGroup.getName());
+
+		_assertAccountGroupAssignments(accountGroup, 1);
+
+		accountGroup =
+			accountGroupResource.getAccountGroupByExternalReferenceCode(
+				"TESTACCOUNTGROUP2");
+
+		Assert.assertNotNull(accountGroup);
+		Assert.assertEquals("Test Account Group 2", accountGroup.getName());
+
+		_assertAccountGroupAssignments(accountGroup, 1);
+
+		accountGroup =
+			accountGroupResource.getAccountGroupByExternalReferenceCode(
+				"TESTACCOUNTGROUP3");
+
+		Assert.assertNotNull(accountGroup);
+		Assert.assertEquals("Test Account Group 3", accountGroup.getName());
+		_assertAccountGroupAssignments(accountGroup, 0);
```

Comment:
```suggestion
		
		_assertAccountGroupAssignments(accountGroup, 0);
```

Reply:
you need to break line like you did before
same for the others in assertAccountGroups2